source/scss/blocks/slider.scss

```
@@ -90,12 +90,26 @@
   }
 }
 
+@keyframes slider-text {
+  from {transform: translateX(100px); opacity: 0}
+  to {transform: translateX(0); opacity: 1}
+}
+
+.swiper-slide-active {
+  .slider__item-text {animation: slider-text .2s ease-in .3s forwards;}
+}
+.swiper-slide-next {
+  .slider__item-text {animation: slider-text .2s ease-in .5s forwards;}
+}
+
```

Можно использовать перечисление селекторов:
```suggestion
.swiper-slide-active,
.swiper-slide-next {
  .slider__item-text {animation: slider-text .2s ease-in .5s forwards;}
}

```